cms/search/publishers.py

```diff
@@ -97,12 +93,17 @@ def _construct_message_for_create_update(self, page: "Page") -> dict:
     def _construct_release_specific_fields(page: "Page") -> dict:
         """Constructs and returns a dictionary with release-specific fields."""
         release_fields = {
+            "provisional_date": None,
             "finalised": page.status in ["CONFIRMED", "PROVISIONAL"],
             "cancelled": page.status == "CANCELLED",
             "published": page.status == "PUBLISHED",
             "date_changes": [],
         }
 
+        # If we do NOT have release_date but we do have release_date_text, treat that as provisional_date
+        if not page.release_date and page.release_date_text:
```

Comment:
Non-blocker, these are mutually exclusive already, there is validation to prevent this already so, you can just check if you have `release_date_text`.